src/FffGcodeWriter.cpp

```diff
@@ -2112,7 +2063,7 @@ void FffGcodeWriter::processSkinInsets(const SliceDataStorage& storage, LayerPla
     // add skin walls aka skin perimeters
     if (extruder_nr == skin_extruder_nr)
     {
-        bool added_something = false;
+        added_something = false;
```

Comment:
Oof, this might fix some unintended travels to 0,0 if this happens to be the only thing printed on a layer. Good find.